File: platform/apps/intel-python/src/intel_service/routes/health.py
"""Health check routes for the intelligence service."""

from fastapi import APIRouter

router = APIRouter(tags=["health"])


@router.get("/health")
async def health_check() -> dict:
    """Unauthenticated health check."""
    return {
        "status": "healthy",
        "service": "cdsip-intel-service",
        "version": "0.1.0",
    }


@router.get("/ready")
async def readiness_check() -> dict:
    """Readiness probe."""
    return {"ready": True}

File: platform/apps/api-rust/src/routes/health.rs
//! Health check endpoint — unauthenticated.

use axum::Json;
use serde_json::{json, Value};

pub async fn health_check() -> Json<Value> {
    Json(json!({
        "status": "healthy",
        "service": "cdsip-api",
        "version": env!("CARGO_PKG_VERSION"),
    }))
}
